tests/components/unifi/test_device_tracker.py

```diff
@@ -560,7 +561,7 @@ async def test_option_track_wired_clients(hass, aioclient_mock, mock_device_regi
     await hass.async_block_till_done()
 
     assert hass.states.get("device_tracker.wireless_client")
-    assert hass.states.get("device_tracker.wired_client")
+    assert not hass.states.get("device_tracker.wired_client")
```

Comment:
The test changes are making me uneasy to merge this refactor. Why do we change the tests? I'd expect them to stay the same to prove that the refactor isn't changing behavior.

Reply:
The test changes are only related to it right now not being possible to automatically load new entities based on changes in config entry options, I keep them like this because I will add them back.

Reply:
I'm not following, what is going to be added back? Changing the tests during a large refactor is really bad. Can't we do it some other way by splitting up the work or doing it in different order?

Reply:
I'm not following, what is going to be added back? Changing the tests during a large refactor is really bad. Can't we do it some other way by splitting up the work or doing it in different order?

Reply:
Previously there was code handling but the current step in refactoring (regarding all the unifi platforms) I have neglected this to tackle that as a future step. This is the last platform to go to the common entity.